### ISSUE TITLE
Document Rooms functionality

### ISSUE THREAD

**Issue by davidarcher** [salesforce/refocus#1212]
I can't find anything in the README, Quickstart, or Doc site about the "Rooms" functionality built into refocus. I presume it's for incident response but it's hard to tell as a new user of Refocus.

**Comment by iamigo** [comment]
Correct. We have no documentation for this functionality yet, but you can get the API docs at <YOUR_REFOCUS_URL>/v1/docs. I'll follow up with the Rooms team to see if we can get them to move some of their internal docs into the github repo.

Thanks!

**Comment by iamigo** [comment]
@cassiodias, let's talk about adding Rooms info to the docs in the github repo.
